src/RunState.cc

```diff
@@ -421,14 +421,19 @@ double check_pseudo_time(const Packet* pkt)
 	return current_pseudo;
 	}
 
+iosource::PktSrc* get_current_pktsrc()
```

Comment:
How about calling this `current_packet_source` to align better with the other `current_*` functions (can't call it `current_pktsrc` unfortunately)?